04-onion-routing.md

```diff
@@ -729,37 +741,38 @@ errors:
        * [`2`:`len`]
        * [`len`:`channel_update`]
 
-The _final node_ MAY, but _intermediate hops_ MUST NOT, return one of the
+The _final node_ MAY, but an _intermediate hop_ MUST NOT, return one of the
 following errors:
   - if the payment hash has already been paid:
     - MAY treat the payment hash as unknown.
-    - MAY succeed in accepting the HTLC.
+    - MAY succeed in accepting the HTLC [FIXME: is this correct?].
```

Comment:
Yeah.  We could say "fail the payment" but it's free money, so...

Reply:
ok didn't change